.github/workflows/build.yml

```diff
@@ -23,11 +23,20 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     
-    - name: dependencies (ubuntu and macos)
-      if: ${{ matrix.os == 'macos-latest' || matrix.os == 'ubuntu-latest' }}
+    - name: dependencies (macos)
+      if: ${{ matrix.os == 'macos-latest' }}
       run: |
+        brew uninstall openssl
+        rm -rf /usr/local/etc/openssl
+        rm -rf /usr/local/etc/openssl@1.1
         brew install pkgconfig doctest
-        brew reinstall openssl
+        brew reinstall openssl@1.1
```

Comment:
Since we just uninstalled it, could this just go up on the `install` line above?